src/index.html

```diff
@@ -85,7 +85,7 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.min.js"></script>
     <script src="https://checkout.stripe.com/checkout.js"></script>
     <script src="/javascript/google-analytics.js"></script>
-    <script src="https://tool.votinginfoproject.org/app.js"></script>
+    
```

Comment:
Hi @lazorfuzz, this polling location finder will be in our new Vote section, so I'd like to keep it in our index.html file for now.